Makefile

```diff
@@ -197,6 +207,10 @@ build-windows:
 	@$(MAKE) build GOOS=windows
 	mv opa_windows_$(GOARCH) opa_windows_$(GOARCH).exe
 
+.PHONY: build-distro-packages
+build-distro-packages:
+	@$(MAKE) deb
+
```

Comment:
Lets remove this and just keep `deb` target for now. Once we iterate and add more targets we can re-add as needed.